doc/content/source/materials/NEMLStress.md

```diff
@@ -36,6 +36,43 @@ cd moose
 This option instructs libMesh to be compiled with the full BOOST installation provided by the `boost` module, rather than the subset of BOOST that is distrubted with libMesh.  
 MOOSE and Blackbear must also be rebuilt.  Rebuilding Blackbear with these changes to include the BOOST library will then automatically build NEML.  
 
+Alternatively, if you are using Conda to manage your MOOSE environment instead of modules, a custom Conda MOOSE environment 
+will need to be created that does not include the prebuilt libMesh library. This Conda environment is set-up with the following commands:
+
+```
+conda create -n moose-boost
+conda activate moose-boost
+conda install moose-petsc moose-tools boost
+```
+
+Once the above Conda environment is set-up, the custom libMesh library containing boost is built using the script and options previously described above.
```

Comment:
Is the libMesh library actually built with boost, or does conda just retrieve a pre-built version with boost? If it's the latter, this needs to be re-worded.

Reply:
Its the former.  You have to build your own libmesh with boost just like when you were using modules.  You also have to make sure you don't create the default conda MOOSE environment that has the prebuilt libmesh.